configurations/dps-payment-service.conf

```diff
@@ -0,0 +1,20 @@
+###############################################################################
+##
+##            DPS Payment Service Specific Environment Variables
+##            Author: @akroon3r
+##
+###############################################################################
+
+SOURCE_CONTEXT_DIR="/src/paymentservice"
+BUILD_CONFIG_TEMPLATE="dps-payment-service.bc.yaml"
+DEPLOY_CONFIG_TEMPLATE="dps-payment-service.dc.yaml"
+APP_NAME="dps-payment-service"
+BUILD_REQUIRED="false"
+SPRING_PROFILES_ACTIVE="cloud,splunk"
+# TODO Change value for all BAMBORA, CRC and Splunk vars as necessary
+BAMBORA_PAYMENT_ENDPOINT="https://example.com"
+CRC_ENDPOINT_APPROVED="http://approved"
+CRC_ENDPOINT_DECLINED="http://declined"
+CRC_ENDPOINT_ERROR="http://error"
+SPLUNK_URL="https://dps-splunk:8088"
+SPLUNK_TOKEN="${SPLUNK_TOKEN}"
```

Comment:
These ones should be set per openshift env

Reply:
removed to be dealt with as secrets later.